Demo/RootViewController.swift

```diff
@@ -41,6 +41,7 @@ final class RootViewController: UIViewController {
     appearance.textInsets = UIEdgeInsets(top: 15, left: 20, bottom: 15, right: 20)
     appearance.bottomOffsetPortrait = 100
     appearance.cornerRadius = 20
+    appearance.shadow = ToastShadow(color: UIColor.black.cgColor, opacity: 0.2, offset: CGSize(width: 0, height: 1), raidus: 10)
```

Comment:
Can we have some switch in the demo to activate/deactivate the shadow? I don't want to demo exclusively with a shadow, as it's not the default behavior.

Reply:
Add 'toggle shadow' to example 1dd2298